src/js/components/Image.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+const METHODS_TO_BIND = ['onImageError'];
+
+class Image extends React.Component {
+  constructor() {
+    super(...arguments);
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  onImageError(event) {
+    let {fallbackSrc} = this.props;
+    if (!fallbackSrc || event.target.src === fallbackSrc) {
+      return;
```

Comment:
What do you think of actually hiding the image in this case? That way we'll never display a broken image.